styles/_utility.css

```diff
@@ -0,0 +1,22 @@
+.img-fluid {
+  height: auto;
+  width: 100%;
+}
+
+figcaption {
+  font-size: 14px;
+  margin-bottom: 20px;
+}
+
+.h1-50 {
```

Comment:
Nicht sicher: Glaube die Klassen werden nie benutzt? Wenn doch, ignorieren.  Würde sie allerdings umbenennen in mb-50